d3a_interface/constants_limits.py

```diff
@@ -19,6 +19,9 @@
 import sys
 from datetime import date, datetime
 from pendulum import duration
+from collections import namedtuple
+
+Ranges = namedtuple('Ranges', ('min', 'max'))
```

Comment:
Trivial, but Ranges is a quite generic name. IMO RangeLimit looks better to my eyes.
PS: you can merge this now, and do the refactoring in the following PR, if you are in a rush